charts/openstack-actions-runner/templates/configmap.yaml

```diff
@@ -6,3 +6,9 @@ data:
   CLOUD_NINE_REGION:  {{ .Values.cloudNineRegion }}
   GITHUB_ORGANIZATION:   {{ .Values.githubOrganization }}
   CLOUD_NINE_TENANT: {{ .Values.cloudNineTenant }}
+  settings.yml: |-
+    github_organization: {{ .Values.githubOrganization }}
+    cloud_nine_region: {{ .Values.cloudNineRegion }}
+    cloud_nine_tenant: {{ .Values.cloudNineTenant }}
+    python_config: {{ .Values.pythonConfigModule }}
+    runner_pool: {{ .Values.runnerPool }}
```

Comment:
You have to set this in a different file as this current configmap is being used to attach environment variables. and `settings.yml` is not an environment variable.